### PR TITLE
#1589 Block/Reject reason

### PR DESCRIPTION
**Pull request by LeraKornienko**
done

### CODE REVIEW

File: src/app/shared/pipes/reason-transform.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'reasonTransform'
```
Comment (litvinets) [inline src/app/shared/pipes/reason-transform.pipe.ts:4]: bad naming, this pipe can be reused for different purposes, so it is better to rename it according to its functionality

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shared/pipes/text-transform.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'textTransform'
```
Comment (litvinets) [inline src/app/shared/pipes/text-transform.pipe.ts:4]: Stillman naming?
You transform text into what?
The naming should reflect the purpose

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — CHANGES_REQUESTED** [review]
you should rename the test file as well, right?

**Comment by litvinets** [comment]
This pipe still does not work correctly, the words are sliced without "-" 
image

**Comment by LeraKornienko** [comment]
link #1632

File: src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/application-card.component.spec.ts [hunk 3]
```diff
@@ -9,11 +9,13 @@ import { MatMenuModule } from '@angular/material/menu';
...
-import { StatusInfoCardComponent } from '../../../../../shared/components/status-info-card/status-info-card.component';
-import { Child } from '../../../../../shared/models/child.model';
 import { JoinPipe } from '../../../../../shared/pipes/join.pipe';
-import { PhoneTransformPipe } from '../../../../../shared/pipes/phone-transform.pipe';
+import { Child } from '../../../../../shared/models/child.model';
+import { TextSliceTransformPipe } from '../../../../../shared/pipes/text-slice-transform.pipe';
+import { StatusInfoCardComponent } from '../../../../../shared/components/status-info-card/status-info-card.component';
 import { InfoStatusComponent } from './info-status/info-status.component';
+import { PhoneTransformPipe } from '../../../../../shared/pipes/phone-transform.pipe';
+
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/application-card.component.spec.ts:18]: remove readundant line

**Review by litvinets — APPROVED** [review]

**Comment by litvinets** [comment]
❤️❤️❤️

**Pull request merged**